utils/python/CIME/XML/build.py

```diff
@@ -1,9 +1,9 @@
 """
```

Comment:
The file is the `xml_file` argument to `write_macros` (line 695 forward). On lines 719-723 we take each `compiler` element and deal with it. Most of the processing of the XML file is in the `CompilerBlock` class (lines 486-648). None of the rest of the file uses `ElementTree` or does any XML processing at all.